src/huggingface_hub/keras_mixin.py

```diff
@@ -27,6 +31,10 @@ def save_pretrained_keras(
         Specify directory in which you want to save the Keras model.
     config (:obj:`dict`, `optional`):
         Configuration object to be saved alongside the model weights.
+    include_optimizer(:obj:`bool`, `optional`):
+        Whether or not to include optimizer in serialization.
+    **model_save_kwargs(:obj:`dict`, `optional`):
```

Comment:
In the docstring you can just put the name, no need for `**`. Same in the other docstring
```suggestion
    model_save_kwargs(:obj:`dict`, `optional`):
```